MonoidalCategories/gap/CoclosedMonoidalCategoriesMethodRecord.gi

```diff
@@ -327,7 +327,30 @@ IsomorphismFromInternalCoHomToObjectWithGivenInternalCoHom := rec(
 
 ) );
 
-CAP_INTERNAL_ENHANCE_NAME_RECORD( COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD );
+# collect closed and closed operations
```

Comment:
I guess this should be "closed and **co**closed". :)

Reply:
Ah, yes, thanks for noticing!